packages/2d/src/components/Node.ts

```diff
@@ -1453,6 +1610,15 @@ export class Node implements Promisable<Node> {
   private signalByKey(key: string): SimpleSignal<any> {
     return (<Record<string, SimpleSignal<any>>>(<unknown>this))[key];
   }
+
+  private reversedChildren() {
+    const children = this.children();
+    const result: Node[] = [];
+    for (let i = children.length - 1; i >= 0; i--) {
+      result.push(children[i]);
+    }
+    return result;
+  }
```

Comment:
This one could be handled natively. JS has an `array.reverse()` already which is heavily supported. You could use that instead or just let the users call it themselves.

```js
node.children().reverse();
```

Reply:
`reverse` modifies the array in place. It would change the cached array in the signal.
I could do `[...node.children()].reverse()` but that's double the iteration for no reason.

Eventually, we'll change it to `toReversed` but this requires ES2023 and typedoc is holding us back at the moment.